docs/setup/keadm.md

```diff
@@ -9,6 +9,35 @@ Please refer [kubernetes-compatibility](https://github.com/kubeedge/kubeedge#kub
 - Currently support of `keadm` is available for Ubuntu and CentOS OS. RaspberryPi supports is in-progress.
 - Need super user rights (or root rights) to run.
 
+**Note**
```

Comment:
We should avoid add too many info in setup docs, can move it to troubleshooting

Reply:
Done